esterni/piano-di-qualifica/components/valutazioni-miglioramento.tex

```diff
@@ -0,0 +1,56 @@
+    Responsabile & Sono sorte diverse difficoltà da parte del responsabile nella suddivisione bilanciata delle ore tra i membri, provocando sovraccarichi di lavoro. & Il gruppo ha deciso di incrementare l'attenzione nella suddivisione della mole di lavoro ai membri del gruppo. \\
+    Analista & Data l'inesperienza nell'analisi, sono sorti diversi problemi nell'individuazione dei requisiti e dei casi d'uso, causando rallentamenti. & La soluzione intrapresa è stata quella di contattare il proponente cercando quindi di velocizzare l'individuazione dei requisiti, in modo da non ritardare la consegna in RR\@. \\
+    Responsabile & Data la poca attenzione, è stato sbagliato il preventivo di progetto, andando sotto la soglia richiesta. Questo errore ha provocato delle conseguenze giustamente adeguate. & Da quel momento il gruppo ha deciso di attuare delle attività di verifica più consistenti, in modo da rilevare il prima possibili errori inaccettabili come questo. \\
+    Verificatore & La poca esperienza nel campo, ha portato ai verificatori lo studio dei documenti su cui attuare le verifiche, provocando ritardi, oppure mancate correzioni, come quella sopra menzionata. & Il gruppo ha deciso di far prestare molta più attenzione ai verificatori, attuando una politica di ispezione attraverso una check-list. \\
+    Programmatore & Sono sorte molte difficoltà nell'utilizzo delle tecnologie a colpa dell'inesperienza nell'uso di esse. Allungando di molto il tempo di autoapprendimento abbiamo raggiunto un punto di non ritorno decidendo di saltare la technology baseline. & Il gruppo ha deciso di sfruttare meglio il tempo a disposizione cercando di massimizzare le ore produttive. \\
+    Amministratore & A causa della complessa continuous integration realizzata, i membri del gruppo si sono trovati in difficoltà causando rallentamenti e di conseguenza saltando la technology baseline. & La soluzione intrapresa è stata quella di limitare l'implementazione di nuovi strumenti di continuous integration, che provocano ritardi inaccettabili di fronte a tempi apparentemente stretti. \\
+  \end{longtable}
+
+ \newpage
+  \subsection{Valutazioni sugli strumenti di lavoro}
+
+  \rowcolors{2}{lightgray}{white!80!lightgray!100}
+  \begin{longtable}[H]{>{\centering\bfseries}m{4cm} >{\centering\arraybackslash}m{6cm} >{\centering\arraybackslash}m{6cm}}
+    \rowcolor{darkgray!90!}
+    \color{white}{\textbf{Strumento}} & \color{white}{\textbf{Descrizione}} & \color{white}{\textbf{Soluzione}} \\
+    GitHub & Il primo problema che sorge spontaneo a tutti i membri del gruppo è come condividere globalmente il codice che viene sviluppato all'interno del gruppo. & La scelta ricade su GitHub, in quanto è uno dei servizi di hosting per progetti software più utilizzati soprattutto nell'ambito di progetti open-source, ed è un implementazione del \glossario{VCS} Git, garantendo la piena compatibilità tra versione locale e versione remota del codice. \\
```

Comment:
"Il primo problema che sorge spontaneo a tutti i membri del gruppo è come condividere globalmente il codice che viene sviluppato all'interno del gruppo." è molto poetico, ma nella realtà dei fatti non è tanto un problema quanto una cosa da fare. Non siamo sbattuti contro il problema di dover condividere il lavoro in remoto, siamo partiti dal presupposto che questa era una cosa da fare e poi abbiamo scelto lo strumento per farla. E questa è una cosa da scrivere nelle norme, non nel pdq.